.drone.yml

```diff
@@ -1,3 +1,15 @@
+# We override the default clone step to workaround a bug with GitHub (see #3415)
+clone:
+  custom:
+    image: plugins/git
+    commands:
+      - sleep 5s
+      - git init
+      - git remote add origin $DRONE_REMOTE_URL
+      - git fetch --no-tags origin $DRONE_COMMIT_REF
```

Comment:
Are you sure this is the merged PR branch reference, as it was before?

Reply:
@nicolasstucki From the log:
```
+ git fetch --no-tags origin $DRONE_COMMIT_REF
From https://github.com/lampepfl/dotty
 * branch                refs/pull/3432/merge -> FETCH_HEAD
```
http://dotty-ci.epfl.ch/lampepfl/dotty/1227

Reply:
Perfect :)